src/phoenix/server/api/helpers/playground_clients.py

```diff
@@ -281,19 +281,22 @@ def supported_invocation_parameters(cls) -> list[InvocationParameter]:
                 max_value=2.0,
             ),
             IntInvocationParameter(
-                invocation_name="max_tokens",
+                invocation_name="max_completion_tokens",
                 canonical_name=CanonicalParameterName.MAX_COMPLETION_TOKENS,
                 label="Max Tokens",
+                default_value=1024,
```

Comment:
yeah this will depend on the model (this looks like it's client or model family based?) so this paradigm might not be quite right for max_tokens, for openai we can keep this one unset dustin since it's not a slider in the ui unset properly displays as unset.

Reply:
ok, removed this default for OpenAI and o1!